src/main/java/com/diamondfire/helpbot/bot/command/impl/stats/individualized/NamesCommand.java

```diff
@@ -37,21 +38,27 @@ public HelpContext getHelpContext() {
                 .addArgument(
                         new HelpContextArgument()
                                 .name("player|uuid")
-                                .optional()
                 );
     }
     
+    @Override
+    protected ArgumentSet compileArguments() {
+        return new ArgumentSet()
+                .addArgument("player|uuid",
+                        new SingleArgumentContainer<>(new StringArgument()));
+    }
```

Comment:
Make a new UserProfileArgument. Look at how we do it in the AbstractPlayerUUIDCommand argument.
Also, name this something different like "profile". However in the help context it can be kept as "player|uuid".

Additionally, please make this optional just like the abstract player uuid. Just do if value == null get the player profile of the player's nickname. Thanks! :)